### ISSUE TITLE
PlaceAtLocation: Updating the object location

### ISSUE THREAD

**Issue by TobiasWehrum** [dmbfm/unity-ar-gps-location-issues#8]
When updating the object location (according to the docs)[https://docs.unity-ar-gps-location.com/guide/#updating-the-object-location], the object only updates when you haven't reached the maximum number of location updates for this object yet.

I would assume that the object would again get x numbers of location updates for the new position.

(And it would be nice to have a way to teleport the object to the new location instead of SmoothMove-ing it even when I normally want to use SmoothMove for the regular position updates due to getting new location/AR data.)

**Comment by dmbfm** [comment]
Yes, it makes sense to clear the Number of location updates, but I don't think that everyone will want that. I will add a method apart from the Location setter which takes an argument for that. 

As for the smooth move, you are correct, I will figure a way to turn it off when updating the position.

**Comment by TobiasWehrum** [comment]
Thanks!

(Although I'm not sure why anyone would want to set a new position without the object updating at all.)
